site/components/src/main/java/com/visitscotland/brxm/services/ResourceBundleService.java

```diff
@@ -54,16 +54,13 @@ public String getResourceBundle(String bundleName, String key, Locale locale){
     }
 
     public String getSiteResourceBundle(String bundleName, String key, Locale locale){
-
-        //Note: Other sites don't currently support other languages
-        if (!properties.getSiteId().isEmpty()) {
-            String siteBundleName = properties.getSiteId() + "." + bundleName;
-            // TODO: This method throws an exception when the site bundle is not found. This bug should be fixed
-            if (getResourceBundleRegistry().getBundle(siteBundleName) != null) {
-                String value = getResourceBundle(siteBundleName, key, locale, true);
-                if (value != null) {
-                    return value;
-                }
+        if (!properties.getSiteId().isEmpty() && hasSiteResourceBundle(bundleName, locale)) {
+            String value = getResourceBundle(getSiteResourceBundleId(bundleName), key, locale, true);
```

Comment:
Side note, just wanted to flag but absolutely a matter of preference - I personally think separating lines can help with readability and debugging.

```suggestion
	    final String bundleId = getSiteResourceBundleId(bundleName);
            String value = getResourceBundle(bundleId, key, locale, true);
```

*Action not required*